core/src/test/java/org/infinispan/scattered/ScatteredSyncFuncTest.java

```diff
@@ -188,4 +191,19 @@ protected void assertLocalValue(int node, MagicKey key, String expectedValue) {
       assertNotNull(ice);
       assertEquals(expectedValue, ice.getValue());
    }
+
+   @Override
+   public void testMergeFromNonOwner() {
+      // TODO : Add support for ScatteredCaches in functional commands : https://issues.jboss.org/browse/ISPN-8078
+      RuntimeException mergeException = new RuntimeException("hi there");
+
+      try {
```

Comment:
`expectException(CacheException.class, UnsupportedOperationException.class, () -> ...)`. There's non-strict version if this is not strictly `CacheException`, too.